### PR TITLE
rpm,deb: package ceph-kvstore-tool man page

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: debian/ceph-test.install [hunk 1]
```diff
@@ -28,3 +28,4 @@ usr/bin/ceph_tpbench
 usr/bin/ceph_xattr_bench
 usr/lib/ceph/ceph-monstore-update-crush.sh
 usr/share/java/libcephfs-test.jar
+usr/share/mon/man8/ceph-kvstore-tool.8
```
Comment (tchaikov) [inline debian/ceph-test.install]: s/mon/man/

**Comment by Liuchang0812** [comment]
@liewegas @tchaikov I saw that ceph-kvstore-tool is packaged in `ceph-test` (is it?).

**Comment by tchaikov** [comment]
@Liuchang0812  yes.

**Comment by tchaikov** [comment]
superseded by #17387

**Pull request closed without merging**